ccl/changefeedccl: TestChangefeedOnlyInitialScan failed

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://mesolite.cluster.engflow.com/invocations/default/e8f0ea3e-413d-47e6-ab81-98f249618814?testReportRun=3&testReportShard=5&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on release-25.3 @ [6b0cb93c0b1a3c9e03924ae1f157a7fccf86b3e5](https://github.com/cockroachdb/cockroach/commits/6b0cb93c0b1a3c9e03924ae1f157a7fccf86b3e5):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestChangefeedOnlyInitialScan2998251405
    test_log_scope.go:76: use -show-logs to present logs inline
    changefeed_test.go:9530: -- test log scope end --
test logs left over in: outputs.zip/logTestChangefeedOnlyInitialScan2998251405
--- FAIL: TestChangefeedOnlyInitialScan (69.44s)
=== RUN   TestChangefeedOnlyInitialScan/cloudstorage
    helpers_test.go:1238: making server as secondary tenant
    helpers_test.go:1317: making cloudstorage feed factory
--- FAIL: TestChangefeedOnlyInitialScan/cloudstorage (69.44s)
=== RUN   TestChangefeedOnlyInitialScan/cloudstorage/initial_scan_only
    changefeed_test.go:9499: failed to start feed for job 0: pq: internal error while retrieving user account memberships: operation "get-user-session" timed out after 10.001s (given timeout 10s): interrupted during singleflight load-value:/global/-1: context deadline exceeded
--- FAIL: TestChangefeedOnlyInitialScan/cloudstorage/initial_scan_only (39.05s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=5</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-60272

--- a/pkg/ccl/changefeedccl/changefeed_test.go
+++ b/pkg/ccl/changefeedccl/changefeed_test.go
@@ -10125,7 +10125,7 @@ func TestChangefeedOnlyInitialScan(t *testing.T) {
 		for testName, changefeedStmt := range initialScanOnlyTests {
 			t.Run(testName, func(t *testing.T) {
 				sqlDB.Exec(t, `CREATE TABLE foo (a INT PRIMARY KEY)`)
-				sqlDB.Exec(t, `INSERT INTO foo (a) SELECT * FROM generate_series(1, 5000);`)
+				sqlDB.Exec(t, `INSERT INTO foo (a) SELECT * FROM generate_series(1, 500);`)
 				defer func() {
 					sqlDB.Exec(t, `DROP TABLE foo`)
 				}()
@@ -10134,10 +10134,10 @@ func TestChangefeedOnlyInitialScan(t *testing.T) {
 				defer closeFeed(t, feed)
 
 				// Insert few more rows after the feed started -- we should not see those emitted.
-				sqlDB.Exec(t, "INSERT INTO foo VALUES (5005), (5007), (5009)")
+				sqlDB.Exec(t, "INSERT INTO foo VALUES (505), (507), (509)")
 
 				var expectedMessages []string
-				for i := 1; i <= 5000; i++ {
+				for i := 1; i <= 500; i++ {
 					expectedMessages = append(expectedMessages, fmt.Sprintf(
 						`foo: [%d]->{"after": {"a": %d}}`, i, i,
 					))

